packages/devtools/lib/src/inspector/inspector.css

```diff
@@ -47,32 +36,6 @@
     margin-left: 25px;
 }
 
-/*
```

Comment:
I would add these back but add an extra css tag.
Essentially these buttons are "collapse-early" 
We need to collapse them at width 1220px but there are other buttons we can wait to collapse for a while.

Reply:
never mind. looks like you added some nice collapse at various size options so the same effect is achieved